plugins/syslog_logs.yaml

```
@@ -88,8 +95,8 @@ template: |
         {{ if .retain_raw_logs }}
         - id: move_raw_log
           type: move
-          from: body
-          to: body.raw_log
+          from: {{ .parse_to }}
```

I think this should still always be body.
```suggestion
          from: body
```

---

updated